packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -280,7 +280,7 @@ export class AtomicSearchInterface
   }
 
   /**
-   * Initializes the connection with the headless search engine using options for `accessToken` (required), `organizationId` (required), `renewAccessToken`, and `platformUrl`.
+   * Initializes the connection with the headless search engine using options for accessToken (required), organizationId (required), renewAccessToken, organizationEndpoints (recommended), and platformUrl (deprecated).
```

Comment:
nitpik: uses `@deprecated`

Reply:
This function expect a type that is derived from Headless, which is itself already marking `platformUrl` as deprecated. 

So it's already doing what you suggest.